src/internal/operators/isEmpty.ts

```diff
@@ -3,6 +3,60 @@ import { Subscriber } from '../Subscriber';
 import { Observable } from '../Observable';
 import { OperatorFunction } from '../types';
 
+/**
+ * Emits true if the input observable emits any values, or emits false if the
+ * input observable completes without emitting any values.
+ *
+ * <span class="informal">Tells whether any values are emitted by an observable</span>
+ *
+ * ![](isEmpty.png)
+ *
+ * `isEmpty` transforms an Observable that emits values into an Observable that
+ * emits a single boolean value representing whether or not any values were
+ * emitted by the source Observable. As soon as the source Observable emits a
+ * value, `isEmpty` will emit a `true`.  If the source Observable completes having
```

Comment:
I think it would be good if this mentioned that this operator completes right after emitting `false` because it seems like it keeps re emitting source:

> As soon as the source Observable emits a value, `isEmpty` will emit a `false` and complete.

instead of this

> As soon as the source Observable emits a value, `isEmpty` will emit a `true`

Reply:
Changed as recommended